coqide-server.opam

```diff
@@ -1,12 +1,23 @@
-opam-version: "1.2"
+synopsis: "The Coq Proof Assistant"
+description: """
+Coq is a formal proof management system. It provides
+a formal language to write mathematical definitions, executable
```

Comment:
What about putting a description of what coqide-server actually is?

Reply:
Indeed, anyone does volunteer to write a version? [Current OPAM packages have no description]